docker-compose.yml

```diff
@@ -1,53 +1,83 @@
 version: '2'
 services:
 
-  postgres:
-    image: postgres
+  database:
+    image: camptocamp/postgis:9.5
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=secret
+    volumes:
+      - postgresql_data:/var/lib/postgresql/data
 
   elasticsearch:
-   image: elasticsearch
+    image: elasticsearch:5.2
+    volumes:
+      - elasticsearch_data:/usr/share/elasticsearch/data
 
   rabbitmq:
-     image: rabbitmq
+     image: rabbitmq:3.6
+     volumes:
+      - rabbitmq_data:/var/lib/rabbitmq
 
   django:
     image: geonode/django
```

Comment:
Image cannot be named as the base image is derives, or strange problems might occur ...
Proposing renaming to `geonode/geonode`